nntrainer/tensor/blas_neon.cpp

```diff
@@ -834,11 +834,13 @@ void hgemv_transpose(const __fp16 *A, const __fp16 *X, __fp16 *Y, uint32_t M,
     Y32[idx] = beta * Y[idx];
   }
   unsigned int i = 0;
-  for (; M - i >= 8; i += 8) {
-    __fp16 x[8];
+  unsigned int N8 = N & -8;
```

Comment:
For bitwise ops, "clean bits", ```N & ~0x7``` is more readable. (a negative number in an expression with unsigned is... irritating)

Reply:
Good point. Will apply right away.

Reply:
``` (N >> 3) << 3 ``` is also a popular expression for readability.